### ISSUE TITLE
Video track mirroring

### ISSUE THREAD

**Issue by grogou** [livekit/components-js#503]
### Select which package(s) are affected

\@livekit/components-core, \@livekit/components-react

### Describe the bug

I have created room like meet.livekit.io example. But in my case all videos mirrored.

Can you please help me with that?

### Reproduction

I have changed my room config like in meet.livekit.io example but also have problem with mirroring.
Also `setLogLevel('silent')` is not working for me. Whole console is about tracks info or something else related to livekit room.

```
<LiveKitRoom
    serverUrl={import.meta.env.LIVEKIT_HOST}
    token={streamInfo.token}
    connect={true}
    video={true}
    audio={true}
    options={roomOptions}
>
    <VideoConference chatMessageFormatter={formatChatMessageLinks} />
    <DebugMode logLevel={LogLevel.silent} />
</LiveKitRoom>
```

### System Info

```shell
System:
    OS: macOS 13.3.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 87.16 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.12.1 - /usr/local/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 8.19.2 - /usr/local/bin/npm
  Browsers:
    Chrome: 113.0.5672.126
    Firefox: 113.0.2
    Safari: 16.4
  npmPackages:
    livekit-client: ^1.6.5 => 1.6.5
```

### Severity

annoyance

**Comment by grogou** [comment]
In your meet demo, you have rotated video by css? Is it good idea to do it like that? 

Strange and wild.

![image](https://github.com/livekit/components-js/assets/10712881/a29c645f-73c7-438d-8289-eee546c08d39)

**Comment by davidzhao** [comment]
I think we should only flip it if it's both:
* local
* front-facing camera

Currently it'll also mirror video even when it's backwards facing.

**Comment by Ocupe** [comment]
@davidzhao is there a reliable way to detect back-facing cameras?

**Comment by Ocupe** [comment]
I see. 👍
We will definitely implement a solution to stop flipping backward facing cameras.

Does the proposed solution unblock you or solve your problem?

**Comment by grogou** [comment]
For now I am using css to rotate all videos. If there will be solution I will upgrade the code.

Thank you all guys.

**Comment by grogou** [comment]
And what about `setLogLevel('silent')`? It is removing some logs, but there still logs in console.

**Comment by Ocupe** [comment]
@grogou I think the logs you are seeing originate from the `livekit-client' package. You can silence them by doing this:

```js
import { setLogLevel } from 'livekit-client';
setLogLevel('silent');
```

Thank you for bringing this up. We will think about how we can make this more obvious or even handle it for you in the future.
If the problem persists, you can reopen the issue or create a new issue.

**Comment by Ocupe** [comment]
Just for reference: https://github.com/livekit/components-js/pull/521

**Comment by grogou** [comment]
@Ocupe thank you for update, with current version of packages `setLogLevel` only comes from core package and it is not working, I can see logs after silent mode is set for logging.

**Comment by Ocupe** [comment]
@grogou #521 is not merged yet. So currently `setLogLevel` only works for the core package.
